src/Storage.Abstraction/BlobContainerClient.cs

```diff
@@ -0,0 +1,110 @@
+﻿using Azure;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Newtonsoft.Json;
+using Storage.Abstraction.Contract;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Storage.Abstraction
+{
+    public class BlobContainerClient : ContainerClientBase, IBlobContainerClient
+    {
+        private BlobServiceClient serviceClient;
+        public BlobContainerClient(object config) : base(config)
+        {
+            serviceClient = this.GetServiceClient();
```

Comment:
Please be consistent, check lines 41 & 44 no "this " there. Or all with this or all without